wkhtmltopdf-image/src/main/docker/app.py

```diff
@@ -46,7 +56,7 @@ def on_generate_pdf(self, request):
             options = json.loads(request.form.get('options', '{}'))
 
         # Evaluate argument to run with subprocess
-        args = [u'wkhtmltopdf']
+        args = [u'wkhtmltopdf', u'-q']
 
         # Add Global Options
```

Comment:
i really, really, really want the command-line invocation logged -- exactly what is being passed to the utility (minus the stdin of course).